autogen/variables.tf

```diff
@@ -280,3 +280,15 @@ variable "issue_client_certificate" {
   description = "Issues a client certificate to authenticate to the cluster endpoint. To maximize the security of your cluster, leave this option disabled. Client certificates don't automatically rotate and aren't easily revocable. WARNING: changing this after cluster creation is destructive!"
   default     = "false"
 }
+
+variable "database_encryption" {
```

Comment:
We should probably add an example (and, by extension, a test case) for this functionality

Reply:
i can do that but that example will have to have key ring and key created like in:
https://cloud.google.com/kubernetes-engine/docs/how-to/encrypting-secrets.

I think that one example will be enough to test this feature. It is the same for public of private cluster.

Reply:
Adding KMS resources to a Terraform example/test case should be pretty straightforward. I'd recommend adding it.

Reply:
example and test added.

Reply:
Thanks!